server/src/models/UserPersonalization.ts

```diff
@@ -0,0 +1,43 @@
+import { QueryBuilder } from 'objection';
+
+import { Domain } from './Domain';
+import { User } from './User';
+import { withDates } from './utils';
+
+export class UserPersonalization extends withDates(Domain) {
+  id!: string;
+  user_id!: string;
+
+  type!: string | null;
+  value!: string | null;
+
+  user!: User;
+
+  static tableName = 'omscentral_user_personalization';
+
+  static relationMappings = {
+    user: {
+      relation: Domain.HasOneRelation,
```

Comment:
Prefer `Domain.BelongsToOneRelation` instead, since a given personalization belongs to a single user.

Reply:
Per your comment above, my understanding is that Domain.HasOneRelation and Domain.BelongsToOneRelation are equivalent.
However, per example in objection [documentation](https://vincit.github.io/objection.js/guide/relations.html#examples) they are different.
Quoting:
"BelongsToOneRelation: Use this relation when the source model has the foreign key
HasManyRelation: Use this relation when the related model has the foreign key."
However, importantly, since both are one-one relationship, I will switch to Domain.BelongsToOneRelation in this case.